src/components/Footer/Footer.tsx

```diff
@@ -17,7 +17,7 @@ function Footer() {
       <GitHubButton
         href={ORG_URL}
         data-size="large"
-        data-show-count="true"
+        data-show-count={true}
```

Comment:
Weird that I copy and paste this from their website :P did typescript catch the error?

Reply:
Yes